nvdaHelper/vbufBackends/mshtml/node.cpp

```diff
@@ -379,7 +379,7 @@ void MshtmlVBufStorage_controlFieldNode_t::preProcessLiveRegion(const MshtmlVBuf
 		if (this->ariaLiveNode) {
 			this->ariaLivePoliteness = this->ariaLiveNode->ariaLivePoliteness;
 		} else {
-			this->ariaLivePoliteness = nullptr;
+			this->ariaLivePoliteness = L"";

```

Comment:
Annoyingly this would have been captured by the visual studio '/analyze' option.

Reply:
Is there a reason why we can't use this?

Reply:
I think it was because there are too many other problems. We would either need to fix all the existing issues (which are likely false positives), or create a system to filter the noise and only show new issues.